tests/platform_tests/test_reboot.py

```diff
@@ -68,7 +68,7 @@ def check_interfaces_and_services(dut, interfaces, reboot_type = None):
     @param interfaces: DUT's interfaces defined by minigraph
     """
     logging.info("Wait until all critical services are fully started")
-    check_critical_services(dut)
+    wait_critical_processes(dut)
```

Comment:
Is this change intentional? replacing 'check_critical_services' with 'wait_critical_processes'. I see that both functions serve different purposes. 

Same question for all changes related to these 2 functions

Reply:
Yes. The function was renamed in the `tests.common.platform.process_utils.py` module. But their purposes are the same.

Reply:
I see a check_critical_processes also in that file. Its purpose is different from 'wait'. It is to ensure that the processes continue to remain up for a certain duration

Reply:
The `check_critical_processes` function watches the critical services for specified time. Once any of the critical service is not running during the watch period, the script will fail by `pytest_assert`. Just after reboot, not all critical services are running. If we use `check_critical_processes`, the test will always fail unnecessarily.

Reply:
I agree that after reboot, it makes sense to use 'wait' in line 60. but in the teardown module, line 42 should be 'check' because we need to ensure that the processes are still up.

Reply:
Ah, I see, that's a good suggestion! Thanks! I'll updte.